src/woo_publications/api/models/category.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ordered_model.models import OrderedModel
+
+from ..constants import InformatieCategorieChoices
+
+
+class InformatieCategorie(OrderedModel):
+    identifier = models.URLField(_("Identifier"), max_length=255, unique=True)
+    name = models.CharField(_("Name"), max_length=80)
+    name_plural = models.CharField(_("Name plural"), max_length=80)
+    definition = models.TextField(_("Definition"), blank=True)
+    origin = models.CharField(
+        _("Origin"),
+        choices=InformatieCategorieChoices.choices,
+        blank=False,
+        max_length=15,
+    )
+
+    class Meta:
+        verbose_name = _("informatie categorie")
+        verbose_name_plural = _("informatie categorieën")
+        ordering = ("order",)
+
+    def save(self, **kwargs):
+        if not self.origin:
+            self.origin = InformatieCategorieChoices.zelf_toegevoegd
+        return super().save(**kwargs)
+
+    def __str__(self):
+        return self.identifier
```

Comment:
Please return the human readable name instead of a system value

```suggestion
    def __str__(self):
        return self.name
```

Reply:
Is this really the best option? Because we aren't stopping anyone from using the same name twice. Should we add the identifier within brackets after the name when the name is duplicate or something like that?

Reply:
Yes, it's just used for informative messages in the admin like "Succesfully changed the information categorie foobarbaz". The `__str__` method is intended for a quick, recognizable representation of the model instance :)